dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -2047,16 +1561,15 @@ data:
         }
       ],
       "refresh": "1h",
-      "schemaVersion": 37,
-      "style": "dark",
+      "schemaVersion": 39,
       "tags": [],
       "templating": {
         "list": [
           {
             "current": {
-              "selected": true,
+              "selected": false,
               "text": "rhtap-observatorium-production",
-              "value": "rhtap-observatorium-production"
+              "value": "P22466E8E7855F1E0"
```

Comment:
This change looks unintentional. Please ensure the prod datasource is selected by default.

Reply:
I'm not sure why that's happening, it seems that Grafana's doing that automatically whenever I export without even touching the datasource.

Anyhow, I've reverted the change